Makefile

```diff
@@ -18,6 +18,11 @@ CLOUDCAFE ?= $(shell which cafe-runner)
 
 .PHONY: targets env-precheck
 
+hooks:
+	echo "#!/bin/bash" > .git/hooks/pre-commit
+	echo "python `pwd`/scripts/config_check.py" >> .git/hooks/pre-commit
```

Comment:
You don't need to interpolate ``` `pwd` ``` here; git hooks are run from the root of the working tree.  (I can't find an official doc reference for this but experimentally the versions of git that I tried on ubuntu OS X, etc, did so.)